src/BlazorUI/Bit.BlazorUI.Extras/Components/MarkdownViewer/BitMarkdownViewer.scss

```
@@ -3,8 +3,166 @@
 .bit-mdv {
     all: revert;
     width: 100%;
+    color: $clr-fg-pri;
+    line-height: 1.6;
+    word-wrap: break-word;
```

Stylelint (17.13.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)